hack/test-cmd.sh

```diff
@@ -89,7 +110,7 @@ for test in "${tests[@]}"; do
 
   os::test::junit::declare_suite_start "cmd/${namespace}-namespace-setup"
   # switch back to a standard identity. This prevents individual tests from changing contexts and messing up other tests
-  os::cmd::expect_success "oc login --server='${KUBERNETES_MASTER}' --certificate-authority='${MASTER_CONFIG_DIR}/ca.crt' -u test-user -p anything"
+  os::cmd::try_until_success "oc login --server=${KUBERNETES_MASTER} --certificate-authority ${MASTER_CONFIG_DIR}/server-ca.crt -u test-user -p anything" $(( 160 * second )) 0.25
```

Comment:
Why drop the quotes?

Reply:
Why the wait?

Reply:
Changed it back. Had it waiting originally before adding preflight checks for the api server